tests/python/pytorch/graphbolt/impl/test_ondisk_dataset.py

```diff
@@ -1694,3 +1694,26 @@ def test_OnDiskDataset_load_tasks():
         original_train_set = None
         modify_train_set = None
         dataset = None
+
+
+def test_BuiltinOnDiskDataset():
+    """Test BuiltinOnDiskDataset."""
+    with tempfile.TemporaryDirectory() as test_dir:
```

Comment:
better to move to regression.

Reply:
this `test-only` dataset is dedicated for unit test that makes sure the download and instantiate logic works as expected. The dataset is quite small and expect to finish very soon.
As for regression, I think what we need to offer is make sure all the built-in dataset(such as ogb-ones) can be downloaded and instantiated successfully.
In a word, both are require in CI and regression.